data/market-local/src/main/java/ir/composenews/localdatasource/database/DbConst.kt

```diff
@@ -0,0 +1,4 @@
+package ir.composenews.localdatasource.database
+
+const val TRUE = 1L
```

Comment:
What do you mean by `TRUE` and `FALSE` ?

Reply:
SQLDelight data types follow SQLite types. `Boolean` types must be stored as `Long` in db.